R-packages/evalcast/R/evaluate.R

```diff
@@ -54,7 +54,10 @@ evaluate_predictions <- function(
                       coverage_80 = interval_coverage(alpha = 0.2)),
   backfill_buffer = 10) {
   responses <- all_attr(predictions_cards, "signals") %>%
-    map(~ .x[1, ])
+    map(~ .x[1, names(.x) != "start_day"]) # RJT: I added this part. I don't
+  # think start dates need to match here. Now that I'm allowing start_day to be 
+  # a function, it can depend on the forecast_date, so the next assertion would
+  # (unintentionally) fail in that case.
```

Comment:
Agreed!

Reply:
Importantly, the function `get_target_response` which is called by `evaluate_predictions_single_ahead` to get the ground truth data for evaluation does not rely on the start_day column from `signals`.